### PR TITLE
Scrapd3/186

### PR DESCRIPTION
**Pull request by tom-nook**
Types of changes
----------------

- Bug fix (non-breaking change which fixes an issue)

Description
-----------

Makes Hero Image responsive
Adds photo credit

Providesfor a better experience on non-desktop screen dimensions

Runs as expected, locally.

Checklist
---------

-  [] I have updated the documentation accordingly
-  [] I have written unit tests

Fixes scrapd/scrapdviz#186

### CODE REVIEW

**Comment by tom-nook** [comment]
This PR does not utilize [Response Breakpoints](https://www.responsivebreakpoints.com/) mentioned in the Issue, but it works fine. Let me know if you'd like that site's tech to have been utilized.

File: components/scrapd-home-section/component.js [hunk 1]
```diff
@@ -31,13 +36,17 @@ const ScrapdHomeSection = props => {
...
   }));
 
-  const Text = styled.div({
-    maxWidth: '75%',
-    minWidth: '75%'
-  });
+  const Text = styled('div')`
+    ${mq({
+      maxWidth: ['99%', '80%'],
+      minWidth: ['99%', '80%'],
```
Comment (rgreinho) [inline components/scrapd-home-section/component.js]: I think you should leave the value at `75%` for the desktop mode.

```js
maxWidth: ['99%', '75%'],
minWidth: ['99%', '75%'],
```

That would preserve the images alternating left/right.

File: components/scrapd-home-section/component.js [hunk 2]
```diff
@@ -31,13 +36,17 @@ const ScrapdHomeSection = props => {
...
+      fontSize: ['1em', '1.6em'],
```
Comment (rgreinho) [inline components/scrapd-home-section/component.js]: `1.6em` for the desktop mode seems a bit too big. You should play with other values, but I think `1.25em` should be enough.

Comment (tom-nook) [reply components/scrapd-home-section/component.js]: Totally agree, yikes.

File: components/scrapd-home-section/component.js (hunk 1 above)
Comment (tom-nook) [reply components/scrapd-home-section/component.js]: Completed.

File: components/scrapd-home-section/component.js (hunk 2 above)
Comment (tom-nook) [reply components/scrapd-home-section/component.js]: Completed.

File: components/scrapd-home-section/component.js [hunk 3]
```diff
@@ -22,7 +22,8 @@ const ScrapdHomeSection = props => {
       backgroundColor: 'white',
       fontSize: ['2em', '2.75em'],
       fontWeight: 'bold',
-      color: '#003CFF'
+      color: '#003CFF',
+      padding: '0px 15px'
```
Comment (tom-nook) [inline components/scrapd-home-section/component.js]: Added some padding to the left and right sides. Here's how it looked before:
Screen Shot 2020-01-28 at 8 00 04 PM

**Comment by tom-nook** [comment]
Done, and ready for review.

**Comment by rgreinho** [comment]
Love it!

Yep, you could work on "Visualizing the data" looks on mobile in another PR, no worries.

**Pull request merged**